docs/insomnia/get-started-with-documents.md

```diff
@@ -0,0 +1,122 @@
+
+Error details include the error type, line number, and error message.
+
+![Errors will appear at the bottom of the center panel. The left and right panels will not be able to render.](/assets/images/design-error-message.png)
+_For error specifics, see the bottom of the center panel._
+
+To replicate the error in the screenshot above with the Pet Store API example, delete "tags:" from line 15. You should see the same errors arise. 
+
+Add line 15 back to move forward with this guide.
+
+### Left Panel Sections
+
+In the left-side panel, a menu populates with the sections in the table below. Modify what appears in your left-side panel by clicking on the three-dot button next to INFO. By default, all _defined_ sections are visible, and INFO is always visible.
+
+If a section has no content, the section title will not populate. For example, the Pet Store API specification that we're working with doesn't have SECURITY defined, so SECURITY won't appear in the left-side panel.
+
+All of this content is generated from your specification. If you want to make modifications to the generated left-side panel content, modify your specification.
+
+{:.table .table-striped}
+Section | Description
+----------- | -----------
+INFO | Overview data about the API, including title, description, version, and license. This section is always visible.
+SERVERS | Servers that access requests for the API.
+PATHS | All paths available for the API, with request types listed.
+REQUESTS | API objects with content types. For example, in the Pet Store API, an object is `Pet` and content types for that object are `application/json` and `application/xml`.
+RESPONSES | Listing of structured, expected responses for each request.
+PARAMETERS | Listing of parameters that the API recognizes.
+HEADERS | Header types accepted by the API.
+SCHEMAS | Schemas detailing the input/output of the API.
+SECURITY | Authentication information and options.
+
+Let's move on to the Debug tab, where we'll get to send individual requests.
+
+## Debug Tab
+
+The Debug tab is the second of the three top-level tabs in Documents. Any requests detected from your specification (imported on the Design tab) will appear in this tab, given that the specification doesn't contain fatal errors.
+
+### Autogenerated Requests
+
+If you've imported a properly formatted specification in your Design tab, you'll see a list in the left-side panel of your Debug tab.
+
+![On the Debug tab, you'll see an autogenerated list of requests from the imported specification.](/assets/images/request-listing-document.png)
+_In the left-side panel, you'll see a list of requests generated from your properly formatted specification._
+
+Click on any request. The center panel will populate with the request URL, request type, base URL variable (defined through the OpenAPI specification), and a body (if application).
+
+Click **Send** and see the response in the right-side panel.
```

Comment:
Is it worth linking to the sending requests guide if they want more detail?

Reply:
Yeah, good idea.